webtool/test_api.py

```
@@ -2622,3 +2622,203 @@ def test_export_project_zip_unknown_project_404(client):
+    import errno as _errno
+    echt_rename = os.rename
+
+    def fake(src, dst):
+        if os.path.basename(src).startswith("S1.mp3"):
+            raise OSError(_errno.ENAMETOOLONG, "Dateiname zu lang")
+        return echt_rename(src, dst)
+
+    monkeypatch.setattr(os, "rename", fake)
+    vorher = _bestand(tmp_path)
+
+    with pytest.raises(OSError) as fehler:
```

Ruff (0.16.2)

[warning] `pytest.raises(OSError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)